File: blablamovebackend/src/main/java/fr/polytech/unice/blablamove/teamc/blablamovebackend/model/CityReport.java
package fr.polytech.unice.blablamove.teamc.blablamovebackend.model;

public class CityReport {
    private City city;
    private Long transactionCount;

    public CityReport(City city, Long transactionCount) {
        this.city = city;
        this.transactionCount = transactionCount;
    }

    public City getCity() {
        return city;
    }

    public void setCity(City city) {
        this.city = city;
    }

    public Long getTransactionCount() {
        return transactionCount;
    }

    public void setTransactionCount(Long transactionCount) {
        this.transactionCount = transactionCount;
    }
}

File: adminMarketingUI/src/app/model/city-report.ts
import {City} from './city';

export class CityReport {
  private _city: City;
  private _transactionCount: number;

  constructor(city: City, transactionCount: number) {
    this._city = city;
    this._transactionCount = transactionCount;
  }

  get city(): City {
    return this._city;
  }

  set city(value: City) {
    this._city = value;
  }

  get transactionCount(): number {
    return this._transactionCount;
  }

  set transactionCount(value: number) {
    this._transactionCount = value;
  }
}
